src/main/java/org/radarbase/output/source/StorageNode.kt

```diff
@@ -0,0 +1,24 @@
+package org.radarbase.output.source
+
+import java.nio.file.Path
+import java.time.Instant
+
+sealed interface StorageNode {
```

Comment:
It might be good to add a line of documentation to these interfaces (i.e. StorageIndex, StorageNode, StorageIndexManager)